### ISSUE TITLE
Shortcode parsing issue in Form Actions

### ISSUE THREAD

**Issue by dbjpanda** [elementor/elementor#11023]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description
Shortcodes produced inside form field's advanced section seem missing quotations marks(") which result shortcodes are not getting parsed while sending Email.
Related Issue #11000, #10967, #10932

## Steps to reproduce
1. Create a contact form using Elementor's Form widget
2. Go to advance section of a field. 

<img width="337" alt="Screenshot 2020-04-01 at 1 13 19 AM" src="https://user-images.githubusercontent.com/17483092/78068462-11e9d280-73b6-11ea-8593-66665f6305a7.png">
3. Copy the shortcode and paste it inside the message field of Email Action. 

<img width="331" alt="Screenshot 2020-04-01 at 1 15 09 AM" src="https://user-images.githubusercontent.com/17483092/78068572-4198da80-73b6-11ea-9ec1-8faa45058f38.png">
4. Save and broswe the form to submit a contact-us test. 
5. Check your email. The email contains raw shortcode. 

<img width="316" alt="Screenshot 2020-04-01 at 1 19 21 AM" src="https://user-images.githubusercontent.com/17483092/78068894-d8659700-73b6-11ea-8a51-578dd718210a.png">

## Expected Result
The shortcode should be parsed and the content of the email should be replaced with real value of shortcodes. 

<img width="270" alt="Screenshot 2020-04-01 at 1 20 02 AM" src="https://user-images.githubusercontent.com/17483092/78068958-eb786700-73b6-11ea-9383-895d6fa80da3.png">

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor Pro v2.9.3

Feel free to update

Thanks!

**Comment by santiagogaal** [comment]
Solved add ""       results    [field id="name"]
